lib/teiserver_web/live/account/profile/profile_components.ex

```diff
@@ -97,7 +97,7 @@ defmodule TeiserverWeb.Account.ProfileComponents do
 
           <%= @user.name %> &nbsp;&nbsp;&nbsp;&nbsp;
           <span style="font-size: 0.7em;">
-            Chevron rank: <%= @user.rank || 0 %>
+            Chevron rank: <%= (@user.rank || 0) + 1 %>
```

Comment:
Can you please change the text to `level` ? Because `rank` in database means 0-indexed.

Reply:
Sure, thing! Done :)